NIPostsReload/Presentation/Scenes/NIPostFeedModule/Views/NIPostFeedViewController.swift

```diff
@@ -13,21 +13,48 @@ protocol NIPostFeedView: AnyObject {
 }
 
 final class NIPostFeedViewController: UIViewController {
+    private struct Constant {
+        static let tabList = [
+            PostFeedType.list.title,
+            PostFeedType.grid.title,
+            PostFeedType.gallery.title
+        ]
```

Comment:
Your PostFeedType is already CaseIterable, so do it through the compactMap. 
*If new types are added in the future, everything will be automatic